script/Arcueid/Day 13/13_00_ARC13_1B.txt

```diff
@@ -659,7 +659,7 @@ After all, the eldest son of the Tohno family was alive and well, living with hi
 [sha:c5f58228da431bd051b75c199d54596336e81d96]{
 -- Page 128, Offset 32027.
 -- 遠野槙久によって幽閉されていたシキが、自分として暮らしていた貴方にどんな感情を抱くかは分かり易すぎるぐらいです」
-SHIKI was being kept locked up somewhere by Tohno Makihisa,
+Tohno Makihisa kept SHIKI locked up somewhere,
```

Comment:
Thought it better flows just like how arc says it

Reply:
Not sure I like this one because it makes Makihisa the main subject, which then makes it sound like the rest of the sentence is _also_ about Makihisa when it's really about SHIKI